### PR TITLE
<VariableInput/> - implement `enterText ` UniDriver method for puppeteer

### PR DESCRIPTION
**Pull request by gendeld**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6041

**Comment by shlomitc** [comment]
We have a similar solution in the `RichTextInputArea` driver. can we merge both?
https://github.com/wix/wix-style-react/blob/46325d51ef22bc99db175db6894481e517ca2455/src/RichTextInputArea/RichTextInputArea.uni.driver.js#L6

**Comment by gendeld** [comment]
> We have a similar solution in the `RichTextInputArea` driver. can we merge both?
> https://github.com/wix/wix-style-react/blob/46325d51ef22bc99db175db6894481e517ca2455/src/RichTextInputArea/RichTextInputArea.uni.driver.js#L6

It's a bit different because RichTextInputArea is not a real input, it uses DraftJS (that's why it uses the beforeInput method)

**Comment by shlomitc** [comment]
@gendeld - but isn't `VariableInput` using DraftJS as well?

**Comment by gendeld** [comment]
> @gendeld - but isn't `VariableInput` using DraftJS as well?

You're right, I'm confused, i'm working on the Input unidriver PR simultaneously

File: src/RichTextInputArea/RichTextInputArea.uni.driver.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import ReactTestUtils from 'react-dom/test-utils';
 
 import { statusIndicatorDriverFactory } from '../StatusIndicator/StatusIndicator.uni.driver';
 
-const enterRichTextValue = async (element, value) => {
+export const enterRichTextValue = async (element, value) => {
```
Comment (shlomitc) [inline src/RichTextInputArea/RichTextInputArea.uni.driver.js:6]: Let’s put this in a shared file for unidriver methods. Ideally this should also be in unidriver’s repo but let’s discuss it with them first

**Review by shlomitc — APPROVED** [review]

**Review by EgoziE — APPROVED** [review]

**Pull request merged**